openapi.yaml

```diff
@@ -2269,3 +2939,9 @@ components:
             read:ssps: read your ssps
             write:parties: modify parties in your account
             read:parties: read your parties
+            write:poams: modify poams in your account
+            read:poams: read your poams
+            write:aps: modify assessment plans in your account
+            read:aps: read your assessment plans parties
+            write:ars: modify assessment results in your account
+            read:ars: read your assessment results in your account
```

Comment:
Here and elsewhere, I'd personally prefer to see us use `assessmentPlan`, `assessment-plan`, `assessmentResults`, or `assessment-results` as appropriate. More like Component Definitions than SSPs and POA&Ms.

Thoughts?

Reply:
I like the use of `assessmentPlan` as it will match `componentDefinitions`